src/meta-srv/src/handler/persist_stats_handler.rs

```diff
@@ -33,8 +34,83 @@ impl HeartbeatHandler for PersistStatsHandler {
             return Ok(());
         }
 
-        // TODO(jiachun): remove stats from `acc` and persist to store
+        let stat = match acc.stats.get(0) {
+            Some(stat) => stat,
+            None => return Ok(()),
+        };
```

Comment:
I think we need to persist all stats to the kv store. Some times we will need an average data